napari_allencell_annotator/widgets/ome_zarr_directory_or_file_dialog.py

```diff
@@ -0,0 +1,44 @@
+from PyQt5.QtWidgets import QListView, QAbstractItemView, QTreeView
+from qtpy.QtWidgets import QFileDialog
+from napari_allencell_annotator._style import Style
+
+
+class OmeZarrDirectoryOrFileDialog(QFileDialog):
+    """
+    A custom QFileDialog that allows the user to select files and zarr directories.
+    """
+
+    def __init__(self, parent, title):
+        super().__init__(parent, title)
```

Comment:
type

Reply:
added in https://github.com/aics-int/napari-allencell-annotator/pull/79/commits/5d6c4d38d51fab0f3a1af0cbdb96244fa25b0cf6